### DISCUSSION TITLE
DetailsList control: How to deselect rows after disableSelectionZone is set to true?

### DISCUSSION THREAD

**Discussion by T41WO** [microsoft/fluentui#19455] [Q&A]
The requirement is to disable further selection after N number of rows is selected. This is achieved by using `selection.count`. 

However, user should be able to "deselect" an already selected row to achieve N - 1 rows selected (setting disableSelectionZone  back to false), in other to perhaps select another. 

How is this achievable? 

Thanks

**Comment by ecraig12345** [comment]
Rather than using `disableSelectionZone`, maybe you could try passing in a custom `Selection` object (via the `selection` prop) with an override of `canSelectItem`, which returns false if too many items are selected.
